integration/integrate-with-email.adoc

```diff
@@ -0,0 +1,25 @@
+[id="integrate-using-email"]
+= Integrating with email
+include::modules/common-attributes.adoc[]
+:context: integrate-with-email
+
+toc::[]
+
+[role="_abstract"]
+Configure {product-title} (RHACS) to send alerts to a standard email provider.
+
+You can use email as a notification method by forwarding alerts from {product-title-short} to a standard email provider.
+To forward alerts from the {product-title-short} platform to an email address, you can use either the `Default Recipients` or `Deployment Annotations` parameter.
+
+You can use default recipients to send email to a standard and centralized team. To send email to a specific team who manages the deployment, use `Deployment Annotations`. To manage an audience by namespace, use `Namespace Annotations`.
+
+You can define an audience to notify about policy violations that are associated with a deployment or namespace using annotation keys. If the deployment has an annotation, the annotation's value overrides the default value. If the namespace has an annotation, the namespace's value overrides the default value.
+
+* If a deployment has an annotation key and a defined audience, an email is sent to the audience who is defined by the key.
+* If a deployment does not have an annotation key, the namespace is checked for an annotation key and an email is sent to the defined audience.
+* If no annotation keys exist, an email is sent to the default recipient that is defined in the integration.
+
+:toolname: Email
+include::modules/configure-acs-kube-security-platform.adoc[leveloffset=+1]
+
+include::modules/configure-policy-notifications.adoc[leveloffset=+1]
```

Comment:
I think you need this small edit to pick up the newly added module: 
```suggestion
include::modules/configure-acs-policy-notifications.adoc[leveloffset=+1]
```

Reply:
@tmalove Please delete the file `modules/configure-acs-policy-notifications.adoc`.

Thank you @bobfuru We want to use the existing `configure-policy-notifications` module here, instead of the new file.

Reply:
@gaurav-nelson I made the review comments and squashed the commits. Let me know if you have any other comments and if none I will have this PR merged. Thanks!